src/app/shared/ui-kit/ui-header/ui-header.component.html

```diff
@@ -0,0 +1,20 @@
+<header
+  class="flex w-full items-center justify-around space-x-8 bg-primary p-4"
+>
+  <div>
+    <img src="assets/images/logo.svg" alt="Company Logo" />
+  </div>
+  <nav>
+    <ul class="flex space-x-4">
+      @for (item of navItems; track $index) {
+        <li>
+          <a [routerLink]="item.link" class="text-bright">{{ item.label }}</a>
+        </li>
+      }
+    </ul>
+  </nav>
+  <div class="button-container">
```

Comment:
why do we need this class button-container?

Reply:
It's my habit to add classes right away for future styling. But with Tailwind, we don't need this